### PR TITLE
{lang}[system/system] Julia v1.6.2

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by robert-mijakovic** [comment]
@boegelbot: please test @ generoso

**Comment by robert-mijakovic** [comment]
Test report by @robert-mijakovic
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login01.meluxina.lxp.lu - Linux RHEL 8.3, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/87f2c77f32e2d20ef65b235228c887c0 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/3e92426b03caa2191e59a2b14f632120 for a full test report.

File: easybuild/easyconfigs/j/Julia/Julia-1.6.2-linux-x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+description = "Julia is a high-level, high-performance dynamic programming language for numerical computing"
+
+toolchain = SYSTEM
+
+source_urls = ['https://julialang-s3.julialang.org/bin/linux/x64/%(version_major_minor)s/']
+sources = ['%(namelower)s-%(version)s%(versionsuffix)s.tar.gz']
+checksums = ['3eb4b5775b0df1ad38f6c409e989501ab445c95bcb01ab02bd60f5bd1e823240']
+
+sanity_check_paths = {
+    'files': ['bin/julia', 'include/julia/julia.h', 'lib/libjulia.so'],
```
Comment (verdurin) [inline easybuild/easyconfigs/j/Julia/Julia-1.6.2-linux-x86_64.eb:25]: Shouldn't this use `SHLIB_EXT`?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/j/Julia/Julia-1.6.2-linux-x86_64.eb (hunk 1 above)
Comment (robert-mijakovic) [reply easybuild/easyconfigs/j/Julia/Julia-1.6.2-linux-x86_64.eb]: True. Fixed!

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @robert-mijakovic!

**Pull request merged**